backend/src/main/java/com/gaucho/sos/security/filters/SecurityRequestFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.gaucho.sos.security.filters;
+
+import com.gaucho.sos.domain.User;
+import com.gaucho.sos.repositories.UserRepository;
+import com.gaucho.sos.security.services.JwtService;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
```

Comment:
Aqui também daria pra remover a linha em branco.
```suggestion
import java.io.IOException;
```

Reply:
Alinhado que vamos manter dessa forma.